python_sdk/pyproject.toml

```diff
@@ -193,6 +193,7 @@ exclude = [
     "build",
     "dist",
     "examples",
+    "tests"
```

Comment:
Why do we add "tests" to the list of excluded paths?

Reply:
without it I'm getting 1300 errors from ruff right now when running `inv sdk.format sdk.lint`
one example below but there is a lot of them

Would be good to get to the bottom of it after the migration

```
python_sdk/tests/unit/test_package.py:24:5: S101 Use of `assert` detected
   |
22 |     groups_extras = {dependency for extra in extras.values() for dependency in extra}
23 |
24 |     assert set(all_extras) == groups_extras
   |     ^^^^^^ S101
   |
```